src/jackdaw/specs.clj

```diff
@@ -32,9 +32,11 @@
 ;; Avro serde
 
 (s/def :jackdaw.serde/confluent-avro-serde
-  (s/keys :req-un [::serde-keyword
-                   ::key?
-                   (or ::schema ::schema-filename)]))
+  (s/keys
+    :opt-un [::schema
+             ::schema-filename]
+    :req-un [::serde-keyword
+             ::key?]))
```

Comment:
Maybe worth have the `req-un` before that `opt-un`. I feel I see that most commonly.

Reply:
👍